src/averageAges.js

```diff
@@ -15,11 +15,17 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = people.filter(({ sex, died }) =>
+    century
+      ? sex === 'm' && Math.ceil(died / 100) === century
+      : sex === 'm'
+  );
+
+  const resultMenAverageAge = men.reduce((totalAge, { born, died }) =>
+    totalAge + died - born, 0
+  ) / men.length;
```

Comment:
Do not repeat your self.
Move this code to separate function and use it here and in `calculateWomenAverageAge`